bug: support LTS Node.js

When using the latest LTS Node.js version (v20.11.0) the `npm install` command fails:

```bash
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: @silverhand/essentials@2.8.6
npm ERR! notsup Not compatible with your version of node/npm: @silverhand/essentials@2.8.6
npm ERR! notsup Required: {"node":"^16.13.0 || ^18.12.0 || ^19.2.0","pnpm":"^8.0.0"}
npm ERR! notsup Actual:   {"npm":"10.3.0","node":"v20.11.0"}
```

--- a/.changeset/sharp-vans-collect.md
+++ b/.changeset/sharp-vans-collect.md
@@ -1,0 +1,10 @@
+---
+"@logto/browser": patch
+"@logto/client": patch
+"@logto/react": patch
+"@logto/node": patch
+"@logto/vue": patch
+"@logto/js": patch
+---
+
+Add Node v20 LTS support

--- a/packages/browser/package.json
+++ b/packages/browser/package.json
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@logto/client": "workspace:^2.3.1",
-    "@silverhand/essentials": "^2.6.2",
+    "@silverhand/essentials": "^2.8.7",
     "js-base64": "^3.7.4"
   },
   "devDependencies": {

--- a/packages/client/package.json
+++ b/packages/client/package.json
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@logto/js": "workspace:^3.0.0",
-    "@silverhand/essentials": "^2.6.2",
+    "@silverhand/essentials": "^2.8.7",
     "camelcase-keys": "^7.0.1",
     "jose": "^5.0.0"
   },

--- a/packages/js/package.json
+++ b/packages/js/package.json
@@ -30,7 +30,7 @@
     "prepack": "pnpm build && pnpm test"
   },
   "dependencies": {
-    "@silverhand/essentials": "^2.8.6",
+    "@silverhand/essentials": "^2.8.7",
     "camelcase-keys": "^7.0.1",
     "jose": "^5.0.0"
   },

--- a/packages/node/package.json
+++ b/packages/node/package.json
@@ -39,7 +39,7 @@
   },
   "dependencies": {
     "@logto/client": "workspace:^2.3.1",
-    "@silverhand/essentials": "^2.6.2",
+    "@silverhand/essentials": "^2.8.7",
     "js-base64": "^3.7.4",
     "node-fetch": "^2.6.7"
   },

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@logto/browser": "workspace:^2.2.1",
-    "@silverhand/essentials": "^2.6.2"
+    "@silverhand/essentials": "^2.8.7"
   },
   "devDependencies": {
     "@silverhand/eslint-config": "^5.0.0",

--- a/packages/vue/package.json
+++ b/packages/vue/package.json
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@logto/browser": "workspace:^2.2.1",
-    "@silverhand/essentials": "^2.6.2"
+    "@silverhand/essentials": "^2.8.7"
   },
   "devDependencies": {
     "@silverhand/eslint-config": "^5.0.0",

